Master-Game/World/BaseCollectible.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace MasterGame.World
+{
+    public class BaseCollectible
+    {
+        public TileType Type { get; }
+        public bool Passable { get; }
+        public bool Occupied { get; set; }
+        public int Elevation { get; }
+        public float Damage { get; }
```

Comment:
What will the damage be? Are we assuming this is always going to change the health of the players collecting it? I Guess we can have this be 0 if this is not supposed to change the health of the player.

Reply:
it could be a hidden bomb